docs/beyond_basics/101-168-dvc.rst

```diff
@@ -0,0 +1,1032 @@
+.. _dvc:
+
+Reproducible machine learning analyses: DataLad as DVC
+------------------------------------------------------
+
+Machine learning analysis are complex:
+Beyond data preparation and general scripting, they typically consist of training and optimizing several different machine learning models and comparing them based on performance metrics.
+This complexity can jeopardize reproducibility -- its hard to remember or figure out which model was trained on which version of what data and which has been the ideal optimization.
+But just like any data analysis project, machine learning projects can become easier to understand and reproduce if they are intuitively structured, appropriately version controlled, and if analyses executions are captured with enough (ideally machine-readable and re-executable) provenance.
+
+.. figure:: ../artwork/src/ML.svg
+
+DataLad provides the functionality to achieve this, and :ref:`previous <yoda>` :ref:`sections <containersrun>` have given some demonstrations on how to do it.
+But in the context of machine learning analyses, other domain-specific tools and workflows exist, too.
+One of the most well-known is `DVC (Data Version Control) <https://dvc.org/>`__, a "version control system for machine learning projects".
+This section compares the two tools and demonstrates `workflows for data versioning, data sharing, and analysis execution <https://realpython.com/python-data-version-control/>`_ in the context of a machine learning project with DVC and DataLad.
+While they share a number of similarities and goals, their respective workflows are quite distinct.
+
+The workflows showcased here are based on a `DVC tutorial <https://realpython.com/python-data-version-control/>`__.
+This tutorial consists of the following steps:
+
+- A data set with pictures of 10 classes of objects (`Imagenette <https://github.com/fastai/imagenette>`_) is version controlled with DVC
+- the data is pushed to a "storage remote" on a local path
```

Comment:
```suggestion
- the data content is pushed to a "storage remote" on a local path
```
or (I thought it would be less appropriate since probably only data content is pushed, not entire dataset, but I am not yet there to judge)
```suggestion
- the dataset is pushed to a "storage remote" on a local path
```

Reply:
I would phrase it like this for DataLad, but as far as I understand DVC, they actually copy the data that's added to DVC.

Reply:
This data duplication effect must be noted, with a remark that git annex does the same on crippled file system

Reply:
Done (although further down, in data sharing)